include/kernel.h

```diff
@@ -879,6 +879,19 @@ __syscall int k_thread_stack_space_get(const struct k_thread *thread,
 void k_thread_system_pool_assign(struct k_thread *thread);
 #endif /* (CONFIG_HEAP_MEM_POOL_SIZE > 0) */
 
+/**
+ * @brief Put the current thread to sleep.
+ *
+ * This routine puts the current thread to sleep for @a duration,
+ * specified as a k_timeout_t object.
+ *
+ * @param ms Desired duration of sleep.
```

Comment:
parameter should not be called `ms` anymore.

Reply:
Good point.